src/runtime/gecko/L20n.jsm

```diff
@@ -0,0 +1,5 @@
+import { ResourceBundle } from './resourcebundle';
+
+this.EXPORTED_SYMBOLS = ["ResourceBundle"];
+
+this.ResourceBundle = ResourceBundle;
```

Comment:
I was trying to make space for the future Env singleton for caching purposes. Do you think you will not want to go there?